src/bindings/python/src/compatibility/openvino/requirements-dev.txt

```diff
@@ -1 +1,2 @@
 cython>=0.29.32
+wheel>=0.40.0
```

Comment:
it's here https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/wheel/requirements-dev.txt